### PR TITLE
Fix point symbol on instantiation

### LINKED ISSUE #4042: points layer ignores symbol
## 🐛 Bug

Constructing a points layer with a non-disc marker symbol currently results in the default (disc) marker being used instead.

## To Reproduce

```python

import numpy as np
import napari

data = np.array([[0,0],[4,4],[8,8]])

napari.view_points(data, symbol='ring')

```

img

## Environment

Happens in napari `0.4.13` on both MacOS and Linux. 

Git blame points to commit b5405402a where the following supposedly important line got removed:

https://github.com/napari/napari/blob/9af23fb0ddd8b9804edc84d534095a00b697c41c/napari/_vispy/layers/points.py#L62

So I think it might be an easy fix e.g. for @brisvag,  @alisterburt, et al

### PR DESCRIPTION
**Pull request by brisvag**
# Description
Fix #4042.

The issue was not the removed line (which was on purpose, after the uspream changes to `MarkersVisual`), but that we were missing `symbol` from the stuff that gets updated on `reset()`, which is called at the end of instantiation to initialize everything.

Also, I removed `on_matrix_change` [edit: and `on_blending_change`] because that's already in the super calls and it was being called twice!

## Type of change

- [x] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).

### CODE REVIEW

File: napari/_vispy/layers/points.py [hunk 1]
```diff
@@ -68,6 +68,7 @@ def _on_data_change(self):
         self.reset()
 
     def _on_symbol_change(self):
+        print('asd')
```
Comment (alisterburt) [inline napari/_vispy/layers/points.py]:
```suggestion
```

File: napari/_vispy/layers/points.py [hunk 2]
```diff
@@ -188,9 +189,8 @@ def _on_canvas_size_limits_change(self):
     def reset(self):
         super().reset()
         self._update_text(update_node=False)
-        self._on_blending_change()
+        self._on_symbol_change()
```
Comment (alisterburt) [inline napari/_vispy/layers/points.py:191]:
```suggestion
        self._on_blending_change()
        self._on_symbol_change()
```

**Review by alisterburt — COMMENTED** [review]
thanks for getting to this quickly @brisvag - looks like a print got leftover and that blending mode got removed

File: napari/_vispy/layers/points.py (hunk 2 above)
Comment (brisvag) [reply napari/_vispy/layers/points.py:191]: Forgot to mention that in the description above, but that is also already present in the base layer, so I removed it on purpose! Yup, the print snuck in xD

Comment (alisterburt) [reply napari/_vispy/layers/points.py:191]: okay cool, in that case LGTM!

**Review by alisterburt — APPROVED** [review]
LGTM - approved pending CI

**Comment by brisvag** [comment]
We're stuck woth the windows failures for now, we can probbably merge! Feel free to do it @alisterburt , otherwise I'll do it later :)

**Pull request merged**